doc/sphinx/source/conf.py

```diff
@@ -0,0 +1,38 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# For the full list of built-in configuration values, see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+import os
+import sys
+
+# -- Project information -----------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#project-information
+
+project = "Neo4j Graph Data Science Python Client"
+copyright = "2023, Neo4j"
+author = "Neo4j"
+release = "2023"
+
+# -- General configuration ---------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#general-configuration
+
+# Add the root of the project to the path so that Sphinx can find the Python sources
+sys.path.append(os.path.join(os.path.dirname(__file__), "..", "..", ".."))
+
+extensions = [
+    "sphinx.ext.autodoc",
+    "sphinx.ext.napoleon",
+    # 'sphinx.ext.autosummary',
+]
+
+# napoleon_use_rtype = False
+
```

Comment:
In the docs we also have `self` inside the parameters, is it possible to not show `self`?

Reply:
Not exactly sure, there could be a parameter for autofunction/docstring that hides this. Do we want to hide it?
Will look at this later.

Reply:
Adam and I thought it would be cleaner and also align as in some places we do already hide the self